tests/NSubstitute.Acceptance.Specs/ReceivedCalls.cs

```diff
@@ -247,6 +248,66 @@ public void Check_when_call_was_received_with_value_type_params_array()
             _car.Received().RecordServiceDates(ServiceDates);
         }
 
+        [Test]
+        [TestCase(1, false)]
+        [TestCase(2, true)]
+        [TestCase(3, true)]
+        [TestCase(4, true)]
+        [TestCase(5, true)]
+        [TestCase(6, false)]
+        public void Check_call_was_received_within_a_range_of_times(int numberOfCalls, bool shouldPass) {
+            for (var i = 0; i < numberOfCalls; i++) {
+                _car.Idle();
+            }
+
+            if (shouldPass) {
+                _car.Received(Quantity.Within(2, 5)).Idle();
+            } else {
+                Assert.Throws<ReceivedCallsException>(() => 
+                    _car.Received(Quantity.Within(2, 5)).Idle()
+                );
+            }
+        }
+
+        [Test]
+        public void Throw_when_call_was_not_received_within_a_range_of_times() {
+            _car.Idle();
+            var ex = Assert.Throws<ReceivedCallsException>(() =>
+                _car.Received(Quantity.Within(2, 5)).Idle()
+            );
+            StringAssert.Contains("between 2 and 5 (inclusive) call", ex.Message);
```

Comment:
Should we assert for "(inclusive calls)" (plural) substring?

Reply:
Good catch, I intended to do this but typo'd 👍